From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: FreeBSD Zen Browser build reference
Subject: [PATCH] Include Zen icon resources on BSD platforms

The Zen icon resource manifest was restricted to XP_LINUX. FreeBSD
uses XP_FREEBSD while using the same GTK/Unix theme resources.

Include the BSD platform defines so the Zen icon resources are packaged
for FreeBSD and the other BSD targets.

---
 browser/themes/shared/zen-icons/jar.inc.mn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/browser/themes/shared/zen-icons/jar.inc.mn b/browser/themes/shared/zen-icons/jar.inc.mn
index 0000000..0000000 100644
--- a/browser/themes/shared/zen-icons/jar.inc.mn
+++ b/browser/themes/shared/zen-icons/jar.inc.mn
@@ -1,5 +1,5 @@
-#ifdef XP_LINUX
+#if defined(XP_LINUX) || defined(XP_FREEBSD) || defined(XP_NETBSD) || defined(XP_OPENBSD)

 [ab-CD]
 % resource zen-icons %content/branding/zen-icons/
-- 
2.0.0
